test/evergreen/tsan_warnings.supp

```diff
@@ -21,3 +23,6 @@ race:__tiered_queue_peek_empty
 
 # The ref track code is diagnostic only and allows for races
 race:__ref_track_state
+
+# __wt_configure_method intentionally allows for races between threads to avoid the overhead of taking a lock on each API call.  
+race:__wt_configure_method
```

Comment:
Further details on why I decided to suppress __wt_configure_method in the [commit message here](https://github.com/wiredtiger/wiredtiger/pull/10413/commits/9efe7da04756451841f15717969892bb2d1a8455).